inst/templates/content-reference-topic.html

```diff
@@ -37,7 +37,9 @@ <h2 class="hasAnchor" id="{{slug}}"><a class="anchor" href="#{{slug}}"></a>{{tit
 
     {{#examples}}
     <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
-    <pre class="examples">{{{ . }}}</pre>
+    {{#bundles}}
+    <div class="examples">{{{ . }}}</div>
+    {{/bundles}}
```

Comment:
Is there a way in mustache to avoid this nesting?

Right I had to make it like so so that the anchor for Examples only show if the examples are present:

```json
{
  "examples": {
    "bundles": [
       "...bundle 1...",
       "...bundle 2...",
       "..."
    ]
  }
}
```